src/startupscriptgenerator/src/common/fsvalidation.go

```diff
@@ -60,6 +60,31 @@ func WriteScript(filePath string, command string) {
 	ioutil.WriteFile(filePath, []byte(command), 0755)
 }
 
+// Appends command to a file
+func AppendScript(filePath string, command string) {
+	if command == "" {
+		return
+	}
+	fmt.Println("Appending output script to '" + filePath + "'")
```

Comment:
Let's move this print statement to be after we check that both `filePath` exists on disc and that we're able to open it.

I would also be in favor of re-wording this statement to be something along the lines of:

```go
fmt.Println("Appending provided command to '" + filePath +"'")
```

Reply:
done